### DISCUSSION TITLE
Request to release the TheHive latest version

### DISCUSSION THREAD

**Discussion by vijaytejak** [Security-Onion-Solutions/securityonion#6312] [Unsupported Versions] [answered]
Hi,
The Hive in security onion is 3.5.1 version, can you release the latest Hive version.

Organization, sharing, MITRE TTPs etc. are missing in the Hive

**Comment by dougburks** [comment, marked as answer]
We will not be updating TheHive as they've changed their license:
https://medium.com/strangebee-announcements/thehive-turns-5-and-adopts-a-model-shaped-for-the-future-95f908719c31

We plan to bring case management functionality directly into Security Onion Console as mentioned here:
https://blog.securityonion.net/2020/12/security-onion-2-in-2020-and-2021.html

We are working on our new case management system now. Expect to hear more about it in the coming months!
